functions/Set-FakkuMetadata.ps1

```diff
@@ -14,7 +14,15 @@ function Set-FakkuMetadata {
                 [Switch]$Log,
 
                 [Parameter(Mandatory = $false)]
-                [System.IO.FileInfo]$LogPath = ".\fkulibrary.log"
+                [System.IO.FileInfo]$LogPath = ".\fkulibrary.log",
+
+                # Sets path to chromedriver.exe and WebDriver.dll
+                [Parameter(Mandatory = $false)]
+                [System.IO.FileInfo]$WebDriverPath = "C:\Selenium",
+
+                # Use this to circumvent chromedriver opening a new window everytime when individually setting metadata. Make sure to open Chrome and login to FAKKU beforehand with the --remote-debugging-port argument (--remote-debugging-port=5656 by default)
+                [Parameter(Mandatory = $false)]
+                [Switch]$Remote
```

Comment:
It would make sense to put `$WebDriverPath` and `$Remote` as required parameters into a [parameter set](https://jeffbrown.tech/how-to-write-awesome-functions-with-powershell-parameter-sets/). You probably don't want to force users to have to use selenium especially if they find the setup too difficult.

Reply:
Not needed.